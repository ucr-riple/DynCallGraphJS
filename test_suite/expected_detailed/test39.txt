{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test39.js:15:3:15:31)': [
        'push (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test39.js:15:3:15:31)'
    ],
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test39.js:17:1:17:21)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test39.js:10:3:12:4)'
    ]
}