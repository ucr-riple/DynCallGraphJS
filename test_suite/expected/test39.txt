{ '(test39.js:15:3:15:31)': [ 'push (Native)' ],
  '(test39.js:17:1:17:21)':
   [ '(test39.js:10:3:12:4)' ] }